src/static.css

```diff
@@ -10,51 +10,35 @@
 }
 
 /* Port old CSS variables to new design system tokens */
+:root:not([style*="--chrome"])[style*="--navy"] {
+  --chrome: rgb(var(--navy));
+}
 :root:not([style*="--accent"])[style*="--deprecated-accent"] {
   --accent: rgb(var(--deprecated-accent));
-}
-:root:not([style*="--accent-fg"])[style*="--navy"] {
   --accent-fg: rgb(var(--navy));
-}
-:root:not([style*="--chrome"])[style*="--navy"] {
-  --chrome: rgb(var(--navy));
+  --accent-fg-light: rgb(var(--white-on-dark));
+  --accent-hover: color-mix(in srgb, rgb(var(--deprecated-accent)), rgb(var(--white-on-dark)) 20%);
+  --accent-pressed: color-mix(in srgb, rgb(var(--deprecated-accent)), rgb(var(--white-on-dark)) 40%);
```

Comment:
These colours are always one and two steps, respectively, towards literal white in every built-in Palette.

I cannot find any actual usages of `--accent-hover` to test it, but given that `--accent` and `--chrome-ui` are the same colour, it made sense to me to make `--accent-hover` the same colour as `--chrome-ui-hover` in the absence of a direct test case.